### PR TITLE
Allow httpd to access glance data

### PR DESCRIPTION
**Pull request by kajinamit**
When glance-api is run by httpd + mod_wsgi, httpd needs access to the glance data directory.

Resolves: RDO-310

### CODE REVIEW

**Review by jpichon — CHANGES_REQUESTED** [review]
Thank you for the bug and PR resolving it! There's a type definition missing but LGTM once that's added.

Additionally, 'rdo' files don't get picked up in the tests at the moment. Here's a [commit example to enable Launchpad bugs](https://github.com/redhat-openstack/openstack-selinux/commit/00e8b1269a511ec465bcb64976297e7c85fb13bc), could you make the same two changes to enable tests/rdo* files testing? Apologies about this. I also get tired of bug tracker proliferation.

(I will be away from the computer for the next couple of weeks, but will follow up when I'm back!)

File: os-glance.te
Comment (kajinamit) [reply os-glance.te:102]: Oops. Thanks for catching it. Fixed in the latest version.

**Comment by kajinamit** [comment]
I've fixed the pointed problem and also updated the test command to include `tests/rdo*`. Thanks for pointing these !

**Comment by jpichon** [comment]
> I've fixed the pointed problem and also updated the test command to include `tests/rdo*`. Thanks for pointing these !

Thank you for the fast reply! Just missing an update to [TEST_FILES in check_all ](https://github.com/redhat-openstack/openstack-selinux/blob/master/tests/check_all#L20) as well.

**Comment by kajinamit** [comment]
> > I've fixed the pointed problem and also updated the test command to include `tests/rdo*`. Thanks for pointing these !
> 
> Thank you for the fast reply! Just missing an update to [TEST_FILES in check_all ](https://github.com/redhat-openstack/openstack-selinux/blob/master/tests/check_all#L20) as well.

Sorry for my carelessness. I've fixed it, too.

**Review by jpichon — APPROVED** [review]

**Pull request merged**
